a couple doxygen errors on develop

## Description
A couple things which are fixed in the PR below. I also see some warnings but that's probably just my system using an old version of doxygen or something. Copy-pasted below for reference.

## Example

Fixed:
```
/home/peter/stan/math2/math/stan/math/opencl/kernels/neg_binomial_2_log_glm_lpmf.hpp:195: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/poisson_log_glm_lpmf.hpp:120: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/neg_binomial_2_log_glm_lpmf.hpp:195: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/poisson_log_glm_lpmf.hpp:120: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/neg_binomial_2_log_glm_lpmf.hpp:195: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/poisson_log_glm_lpmf.hpp:120: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/neg_binomial_2_log_glm_lpmf.hpp:195: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/poisson_log_glm_lpmf.hpp:120: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/neg_binomial_2_log_glm_lpmf.hpp:195: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/opencl/kernels/poisson_log_glm_lpmf.hpp:120: warning: unable to resolve link to `kernels/subtract.hpp' for \link command
/home/peter/stan/math2/math/stan/math/prim/meta/get.hpp:41: warning: argument 'x' of command @param is not found in the argument list of stan::get(const T &m, size_t n)
/home/peter/stan/math2/math/stan/math/prim/meta/get.hpp:47: warning: The following parameters of stan::get(const T &m, size_t n) are not documented:
  parameter 'm'
```

Ignored:
```
/home/peter/stan/math2/math/README.md:4: warning: Unexpected html tag <img> found within <a href=...> context
/home/peter/stan/math2/math/README.md:8: warning: unable to resolve reference to `https:' for \ref command
/home/peter/stan/math2/math/doxygen/contributor_help_pages/developer_doc.md:42: warning: unable to resolve reference to `https:' for \ref command
```

## Expected Output
Same, but no doxygen errors.

#### Current Version:
v3.0.0

--- a/stan/math/opencl/kernels/neg_binomial_2_log_glm_lpmf.hpp
+++ b/stan/math/opencl/kernels/neg_binomial_2_log_glm_lpmf.hpp
@@ -193,7 +193,8 @@ static const char* neg_binomial_2_log_glm_kernel_code = STRINGIFY(
 // \endcond
 
 /** \ingroup opencl_kernels
- * See the docs for \link kernels/subtract.hpp subtract() \endlink
+ * See the docs for \link kernels/neg_binomial_2_log_glm_lpmf.hpp
+ * neg_binomial_2_log_glm_lpmf() \endlink
  */
 const kernel_cl<out_buffer, out_buffer, out_buffer, out_buffer, in_buffer,
                 in_buffer, in_buffer, in_buffer, in_buffer, int, int, int, int,

--- a/stan/math/opencl/kernels/poisson_log_glm_lpmf.hpp
+++ b/stan/math/opencl/kernels/poisson_log_glm_lpmf.hpp
@@ -118,7 +118,8 @@ static const char* poisson_log_glm_kernel_code = STRINGIFY(
 // \endcond
 
 /** \ingroup opencl_kernels
- * See the docs for \link kernels/subtract.hpp subtract() \endlink
+ * See the docs for \link kernels/poisson_log_glm_lpmf.hpp
+ * poisson_log_glm_lpmf() \endlink
  */
 const kernel_cl<out_buffer, out_buffer, out_buffer, in_buffer, in_buffer,
                 in_buffer, in_buffer, int, int, int, int, int, int>

--- a/stan/math/prim/meta/get.hpp
+++ b/stan/math/prim/meta/get.hpp
@@ -39,7 +39,7 @@ inline T get(const std::vector<T>& x, size_t n) {
 /** \ingroup type_trait
  * Returns the n-th element of the provided Eigen matrix.
  *
- * @param x input \c Eigen \c Matrix or expression
+ * @param m input \c Eigen \c Matrix or expression
  * @param n index of the element to return
  * @return n-th element of the \c Eigen \c Matrix or expression
  */

